tests/baselines/reference/tsxReactEmitEntities.js

```diff
@@ -12,5 +12,5 @@ declare var React: any;
 
 
 //// [file.js]
-React.createElement("div", null, "Dot goes here: · &notAnEntity; ");
+React.createElement("div", null, "Dot goes here: \u00B7 &notAnEntity; ");

```

Comment:
I will accept this baseline for now, but may revisit as we discuss whether we need to call `escapeNonAsciiCharacters` for every synthetic literal. It may only really be useful for performing string canonicalization for module import deduplication.